boot/configure.ml

```diff
@@ -90,5 +98,7 @@ let () =
   pr "  ; sbin = %s" (option string !sbindir);
   pr "  ; libexec_root = %s" (option string !libexecdir);
   pr "  }";
+  pr "";
+  pr "let init () = %s" (toggle_inits !toggles);
```

Comment:
It would be nice if typos in config variable names or attempts to enable non-existent variables could be caught at compile time (currently these cases are ignored if I understand correctly). One way to do this might be be introducing a variant type for the config options and requiring the user to pass the constructor names to `--toggles`. The constructors could also be documented in the mli file avoiding the issue for users to look inside the ml file to use this feature.

Reply:
This should be resolved now, but I did it the way Rudi suggested, with `--enable-toolchains`.
So if someone tries `--enable-doesntexist` or `--enable-tolochains`, the error is going to be picked up